src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,113 +1,85 @@
-  // objects
-  public enum intakePosition {
-    INTAKE_CLOSED, 
-    INTAKE_OPEN
-  }
-
-// Map of intake positions
-EnumMap<intakePosition, Double> intakePositions = new EnumMap<>(intakePosition.class);
-
-// variable to set the arm position when scoring and intaking
-private intakePosition currentIntakePosition = intakePosition.INTAKE_CLOSED;
+private double intaking;
+private DigitalInput sensor;
 
 public Intake() {
 
   intakeMotor = new CANSparkMax(Constants.Intake.kIntakeID, MotorType.kBrushless);
+  sensor = new DigitalInput(0);
 
   // set intake motor to factory defaults for if we ever want to switch them out 
   intakeMotor.restoreFactoryDefaults();
 
-  //intakeEncoder = intakeMotor.getAbsoluteEncoder(Type.kDutyCycle);
-  intakeRelativeEncoder = intakeMotor.getEncoder();
-
   // set intake basic values 
   intakeMotor.setSmartCurrentLimit(Constants.Intake.kIntakeCurrentLimit);
   intakeMotor.setInverted(Constants.Intake.kIntakeInverted);
   intakeMotor.setIdleMode(Constants.Intake.kIntakeIdleMode);
 
-  // set the intake PID controllers
-  intakePIDController = intakeMotor.getPIDController();
-  // intakePIDController.setFeedbackDevice(intakeEncoder);
-  intakePIDController.setFeedbackDevice(intakeRelativeEncoder);
-  intakePIDController.setP(Constants.Intake.kIntakePIDGains.P);
-  intakePIDController.setOutputRange(
-      Constants.Intake.kMinOutput, Constants.Intake.kMaxOutput);
-
-  // enable and set intake motor forward and reverse limit
-  intakeMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
-  intakeMotor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
-  intakeMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, Constants.Intake.kFLimit);
-  intakeMotor.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, Constants.Intake.kRLimit);
-
   intakeMotor.burnFlash();
 
-  intakePositions.put(intakePosition.INTAKE_OPEN,  Constants.Intake.kIntakeOpen);
-  intakePositions.put(intakePosition.INTAKE_CLOSED,  Constants.Intake.kIntakeClosed);
-
-  setDefaultCommand(new RunCommand(this::idle, this));
+  setDefaultCommand(new RunCommand(this::stop, this));
 }
 
+
+// Returns the speed of the NEO550 motor
 public double getSpeed() {
-  return intakeMotor.getAppliedOutput();
+  return this.intakeMotor.get();
 }
 
-// returns the raw position of the intake encoder in rotations
-public double getPosition() {
-  return intakeEncoder.getPosition();
+// sets the intake motor to 0 to stop all movement 
+public void idle(double speed) {
+  intakeMotor.set(speed);
 }
 
-// // returns the absolute rotation of the intake Absolute Encoder without the offset
-// public Rotation2d getAbsoluteRotation() {
-//   return new Rotation2d(getPosition() - Constants.Intake.kIntakeOffset);
-// }
-
-// sets the intake motor to 0 to stop all movement 
-public void idle() {
+public void stop(){
   intakeMotor.set(0.0);
 }
 
-public void MunaulPickup(){
```

Comment:
Misspelled manual

Reply:
the code has been deleted by someone else